internal/commands/ip_access/create_inputs.go

```diff
@@ -0,0 +1,22 @@
+package ipaccess
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/terminal"
+)
+
+type createInputs struct {
+	cli.ProjectInputs
+	IPAddress  string
+	Comment    string
+	UseCurrent bool
```

Comment:
what are `UseCurrent` or `AllowAll` used for?

Reply:
`UseCurrent` uses the users current IP address during creation. `AllowAll` allows access from anywhere (i.e `0.0.0.0/0`)

Reply:
Just got this resolved, had to change/update the SDK as well. Thanks for pointing it out!